docs/reference/search/search-template.asciidoc

```diff
@@ -32,7 +32,209 @@ disable scripts per type and context as described in the
 <<allowed-script-types-setting, scripting docs>>
 
 [float]
-==== More template examples
+==== Examples
+
+[float]
+[[pre-registered-templates]]
+===== Store a search template
+
+You can store a search template using the stored scripts API.
+
+[source,js]
+------------------------------------------
+POST _scripts/<templatename>
```

Comment:
I believe we've used the terminology "id" both in code and api for stored scripts. We should stay consistent here and call this `templateid`?

Reply:
Good call! Fixed with 71d03edd885.